src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData.cs

```diff
@@ -788,6 +788,24 @@ public DisplayEntry(string value, DisplayEntryValueType type)
             ValueType = type;
         }
 
+        /// <summary>
+        /// Creates a DisplayEntry for the given scriptblock
+        /// </summary>
+        /// <param name="scriptblock">The content of the scriptblock</param>
+        public static DisplayEntry CreateScriptBlockEntry(string scriptblock)
```

Comment:
Where we use the method?

Reply:
We don't. 
But `CreatePropertyEntry` is used now, so I added the scriptblock one for completeness.

Reply:
Clear. 
We should add tests for the public method.